.pre-commit-config.yaml

```diff
@@ -15,15 +15,26 @@ repos:
   - id: check-added-large-files
   - id: check-merge-conflict
   - id: check-yaml
+  - id: debug-statements
   - id: end-of-file-fixer
   - id: mixed-line-ending
     name: Force line endings to LF
     args: ['--fix=lf']
   - id: trailing-whitespace
 
 - repo: https://github.com/pre-commit/pygrep-hooks
-  rev: v1.9.0
+  rev: v1.10.0
```

Comment:
could dependabot do this for us?

Reply:
I don't think so as this isn't a GitHub Action nor one of the other package ecosystems dependabot supports. Pre-commit's hosted CI can do this, but we are currently stuck with v2.5.0 for the other set of hooks while we're still supporting Python 2.